### PR TITLE
Add installation sections on Windows for winget and standalone

### PR DESCRIPTION
**Pull request by Zaid-Ajaj**
After [Pulumi #4676](https://github.com/pulumi/pulumi/issues/4676) was resolved, it is now possible to install Pulumi using `winget` on Windows but also using a standalone Windows x64 installer. This PR extends the installation section to include these options. 

Also makes the distinction for manual installation where "manual" means downloading binaries to distinguish from downloading the standalone installer

### CODE REVIEW

File: themes/default/content/docs/get-started/install/_index.md
Comment (susanev) [inline themes/default/content/docs/get-started/install/_index.md:172]: these numbers will automatically change to 1, 2, 3 form 1. 1. as part of rendering. and leaving it as 1, 1, etc can make it easier to add future items without adjusting all the following number.

**Review by susanev — APPROVED** [review]

File: themes/default/content/docs/get-started/install/_index.md
Comment (Zaid-Ajaj) [reply themes/default/content/docs/get-started/install/_index.md:172]: I looked at the live version at https://www.pulumi.com/docs/get-started/install/ and there the numbers don't look like they are ordered 1, 2, 3. Instead they are numbered 1, 1, 2 🤔

Comment (susanev) [reply themes/default/content/docs/get-started/install/_index.md:172]: ah yah the markdown lint comments messes with that functionality, ignore me then!

File: themes/default/content/docs/get-started/install/_index.md
Comment (cnunciato) [inline themes/default/content/docs/get-started/install/_index.md:134]: Super minor, but should this have an article before it? (Seems like either "using winget" or "using the winget package manager" would work, unless "winget package manager" is the actual proper name.)

```suggestion
Install Pulumi using the [winget-cli](https://github.com/microsoft/winget-cli/) package manager. This is built-in on Windows 11 and later.
```

**Review by cnunciato — APPROVED** [review]
One thing to consider. Thanks! Feel free to merge when you like.

**Comment by Zaid-Ajaj** [comment]
Thanks @cnunciato for the suggestion, it makes sense. I applied the change and will merge when the CI goes green 😄

File: themes/default/content/docs/get-started/install/_index.md
Comment (lukehoban) [inline themes/default/content/docs/get-started/install/_index.md:132]: Curious - should this be listed first?  I'm not sure of the ecosystem usage of Winget vs. Chocolatey, but my sense is that Winget is more "official" these days.  Is there any reason from our perspective why we would expect folks to have a better experience with Chocolatey?

Comment (cnunciato) [reply themes/default/content/docs/get-started/install/_index.md:132]: Relatedly, if that were the case, would it makes sense to update the Get Started tutorials to use Winget also?

Comment (Zaid-Ajaj) [reply themes/default/content/docs/get-started/install/_index.md:132]: > Is there any reason from our perspective why we would expect folks to have a better experience with Chocolatey?

@lukehoban winget is indeed the "more official" package manager but my assumption is that Windows users are more familiar with chocolatey as it has been used and known for a much longer time than winget (which went [v1.0](https://devblogs.microsoft.com/commandline/windows-package-manager-1-0/) around 05/2021)

Another _catch_ of winget support is that pulumi releases are not immediately published to the winget registry: automated PRs such as [this one](https://github.com/microsoft/winget-pkgs/pull/56531) are sent to the official registry and have to be reviewed and merged by a maintainer so users might not always install latest pulumi because of this delay 

If ask me personally, I think the "Standalone Installer" could be the _default_ because it doesn't require any package manager (chocolatey, winget, etc.), it's just an MSI which means people can just uninstall it from control panel if they want and it is immediately created in [Releases of pulumi-winget](https://github.com/pulumi/pulumi-winget/releases) as soon as pulumi publishes a new version.

Comment (mikhailshilkov) [reply themes/default/content/docs/get-started/install/_index.md:132]: > I think the "Standalone Installer" could be the default because it doesn't require any package manager

We don't have any auto-upgrade functionality, so in this sense it may be more beneficial to push users towards package managers than a manual installation. My 2c.

Comment (Zaid-Ajaj) [reply themes/default/content/docs/get-started/install/_index.md:132]: > We don't have any auto-upgrade functionality

@mikhailshilkov That's a great point, these installers don't have that indeed.

**Pull request merged**
